### DISCUSSION TITLE
Custom keyboard shortcuts for all addons

### DISCUSSION THREAD

**Discussion by bobby-sills** [ScratchAddons/ScratchAddons#4931] [Ideas] [answered]
I ran into a problem with scratch addons. When I tried to use the dev tools block search bar (which is enabled by pressing cmd-space) it would trigger spotlight search. You could fix this by making all the addons with keyboard shortcuts editable.

A way that it could look:
<img width="811" alt="Screen Shot 2022-07-31 at 12 53 21 PM" src="https://user-images.githubusercontent.com/76852105/182042838-f4fa6e25-4999-4052-a89c-c108f43b5dbc.png">
 I think this would be quite useful, please add it.

**Comment by WorldLanguages** [comment, marked as answer]
We plan to make all hotkeys customizable in the future, including Scratch hotkeys (like Ctrl+S)

**Comment by rimopa** [comment]
if you use Chrome you can go to chrome://extensions/shortcuts and configure it.

**Reply by WorldLanguages** [reply]
That's for the extension popup, not the devtools popup.
